<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1354</width>
    <height>563</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>255</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>94</red>
        <green>210</green>
        <blue>76</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>255</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>94</red>
        <green>210</green>
        <blue>76</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>255</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>94</red>
        <green>210</green>
        <blue>76</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>94</red>
        <green>210</green>
        <blue>76</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Nutrition plan</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>841</width>
      <height>401</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <layout class="QHBoxLayout" name="all">
     <item>
      <layout class="QVBoxLayout" name="menu1">
       <item>
        <widget class="QLabel" name="name1">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Alex</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="b1">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="l1">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="d1">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="s1">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="menu2">
       <item>
        <widget class="QLabel" name="name2">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Clair</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="b2">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="l2">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="d2">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="s2">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="menu3">
       <item>
        <widget class="QLabel" name="name3">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Tori</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="b3">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="l3">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="d3">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="s3">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="menu4">
       <item>
        <widget class="QLabel" name="name4">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="b4">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="l4">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="d4">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="s4">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="menu5">
       <item>
        <widget class="QLabel" name="name5">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="b5">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="l5">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="d5">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="s5">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="menu6">
       <item>
        <widget class="QLabel" name="name6">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="b6">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="l6">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="d6">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="s6">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="shown">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>430</y>
      <width>541</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Slab</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">float:right;
margin-right: 3%;
margin top: 5px;
border-radius:5px;
-moz-border-radius:5px;
webkit-border-radius:5px;
padding:5px 9px;
font-size: 1.2em;
background-color:rgba(85, 184, 65,1);
text-shadow: #454545 0 0 2px;
border-bottom: 4px solid rgba(98, 193, 71, 1);
color:black;
font-family: 'Roboto Slab', senif;</string>
    </property>
    <property name="text">
     <string>Показати відповідність вмісту нутріентів у дієті</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>151</width>
      <height>402</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <layout class="QVBoxLayout" name="dishes">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Страви</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="breakfast">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="lunch">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="dinner">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="snacs">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>191</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Оберіть день</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="monday">
       <property name="font">
        <font>
         <family>Roboto Slab</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">float:right;
margin-right: 3%;
margin top: 5px;
border-radius:5px;
-moz-border-radius:5px;
webkit-border-radius:5px;
padding:5px 9px;
font-size: 1.2em;
background-color:rgba(85, 184, 65,1);
text-shadow: #454545 0 0 2px;
border-bottom: 4px solid rgba(98, 193, 71, 1);
color:black;
font-family: 'Roboto Slab', senif;</string>
       </property>
       <property name="text">
        <string>Понеділок</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tuesday">
       <property name="font">
        <font>
         <family>Roboto Slab</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">float:right;
margin-right: 3%;
margin top: 5px;
border-radius:5px;
-moz-border-radius:5px;
webkit-border-radius:5px;
padding:5px 9px;
font-size: 1.2em;
background-color:rgba(85, 184, 65,1);
text-shadow: #454545 0 0 2px;
border-bottom: 4px solid rgba(98, 193, 71, 1);
color:black;
font-family: 'Roboto Slab', senif;</string>
       </property>
       <property name="text">
        <string>Вівторок</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="wednesday">
       <property name="font">
        <font>
         <family>Roboto Slab</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">float:right;
margin-right: 3%;
margin top: 5px;
border-radius:5px;
-moz-border-radius:5px;
webkit-border-radius:5px;
padding:5px 9px;
font-size: 1.2em;
background-color:rgba(85, 184, 65,1);
text-shadow: #454545 0 0 2px;
border-bottom: 4px solid rgba(98, 193, 71, 1);
color:black;
font-family: 'Roboto Slab', senif;</string>
       </property>
       <property name="text">
        <string>Середа</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="thursday">
       <property name="font">
        <font>
         <family>Roboto Slab</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">float:right;
margin-right: 3%;
margin top: 5px;
border-radius:5px;
-moz-border-radius:5px;
webkit-border-radius:5px;
padding:5px 9px;
font-size: 1.2em;
background-color:rgba(85, 184, 65,1);
text-shadow: #454545 0 0 2px;
border-bottom: 4px solid rgba(98, 193, 71, 1);
color:black;
font-family: 'Roboto Slab', senif;</string>
       </property>
       <property name="text">
        <string>Четвер</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="friday">
       <property name="font">
        <font>
         <family>Roboto Slab</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">float:right;
margin-right: 3%;
margin top: 5px;
border-radius:5px;
-moz-border-radius:5px;
webkit-border-radius:5px;
padding:5px 9px;
font-size: 1.2em;
background-color:rgba(85, 184, 65,1);
text-shadow: #454545 0 0 2px;
border-bottom: 4px solid rgba(98, 193, 71, 1);
color:black;
font-family: 'Roboto Slab', senif;</string>
       </property>
       <property name="text">
        <string>П'ятниця</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saturday">
       <property name="font">
        <font>
         <family>Roboto Slab</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">float:right;
margin-right: 3%;
margin top: 5px;
border-radius:5px;
-moz-border-radius:5px;
webkit-border-radius:5px;
padding:5px 9px;
font-size: 1.2em;
background-color:rgba(85, 184, 65,1);
text-shadow: #454545 0 0 2px;
border-bottom: 4px solid rgba(98, 193, 71, 1);
color:black;
font-family: 'Roboto Slab', senif;</string>
       </property>
       <property name="text">
        <string>Субота</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sunday">
       <property name="font">
        <font>
         <family>Roboto Slab</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">float:right;
margin-right: 3%;
margin top: 5px;
border-radius:5px;
-moz-border-radius:5px;
webkit-border-radius:5px;
padding:5px 9px;
font-size: 1.2em;
background-color:rgba(85, 184, 65,1);
text-shadow: #454545 0 0 2px;
border-bottom: 4px solid rgba(98, 193, 71, 1);
color:black;
font-family: 'Roboto Slab', senif;</string>
       </property>
       <property name="text">
        <string>Неділя</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сніданок</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Обід</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Вечеря</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>330</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Перекус</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shown1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>430</y>
      <width>541</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Slab</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">float:right;
margin-right: 3%;
margin top: 5px;
border-radius:5px;
-moz-border-radius:5px;
webkit-border-radius:5px;
padding:5px 9px;
font-size: 1.2em;
background-color:rgba(85, 184, 65,1);
text-shadow: #454545 0 0 2px;
border-bottom: 4px solid rgba(98, 193, 71, 1);
color:black;
font-family: 'Roboto Slab', senif;</string>
    </property>
    <property name="text">
     <string>Показати дані користувача</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>430</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Slab</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">float:right;
margin-right: 3%;
margin top: 5px;
border-radius:15px;
-moz-border-radius:5px;
webkit-border-radius:5px;
padding:5px 9px;
font-size: 1.2em;
background-color:rgba(85, 184, 65,1);
text-shadow: #454545 0 0 2px;
border-bottom: 4px solid rgba(98, 193, 71, 1);
color:black;
font-family: 'Roboto Slab', senif;</string>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1354</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
